test/integration/targets/plugin_loader/ignore_exts/ignore_exts.yml

```diff
@@ -0,0 +1,18 @@
+- hosts: testhost
```

Comment:
Should probably add some version of this test to the `collections` integration test target as well, since this is only testing the legacy side of it.